apps/indexer/src/api/services/proposals/proposals.ts

```diff
@@ -104,4 +119,47 @@ export class ProposalsService {
 
     return { ...proposal, status };
   }
+
+  /**
+   * Returns the delegates with active delegations that didn't vote on a given proposal
+   */
+  async getProposalNonVoters(
+    proposalId: string,
+    skip: number = 0,
+    limit: number,
+    orderDirection: "asc" | "desc",
+    addresses?: Address[],
+  ): Promise<VotersResponse> {
+    const nonVoters = await this.proposalsRepo.getProposalNonVoters(
+      proposalId,
+      skip,
+      limit,
+      orderDirection,
+      addresses,
+    );
+
+    const _addresses = addresses ? addresses : nonVoters.map((v) => v.voter);
+
+    const comparisonTimestamp = Math.floor(Date.now() / 1000 - DaysEnum["30d"]);
```

Comment:
"DaysEnum["30d"]" smells strange uheuehuheueh. Is it correct? why do we have it?

Reply:
30 days in seconds, not beautiful but it works